charts/k8ssandra-cluster/templates/stargate/service.yaml

```diff
@@ -1,4 +1,3 @@
----
```

Comment:
the linter doesn't like these.

Reply:
What is the line that causes the problem, and what is the error?

Reply:
I don't think it gave a line number; it said that the object didn't have a name. I believe that the divider causes the linter to see an implicit "object" before it, with no properties at all.